bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormComponent.java

```diff
@@ -76,6 +76,12 @@ default Map<String, Object> getDorProperties() {
         return Collections.emptyMap();
     }
 
+    @JsonView(Views.Author.class)
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    default Map<String, Object> getSignerProperties() {
```

Comment:
Please add java doc for these changes, also add since in the java doc as this is a new API

Reply:
Done